src/lib/core.js

```diff
@@ -18,7 +17,12 @@ class Core extends EventEmitter {
     super()
     this.clientList = [] // Client[]
     this.clients = {} // { prefix ⇒ Client }
-    this.tables = options.routingTables || new RoutingTables(null, [], null)
+    this.tables = options.routingTables || new RoutingTables([], null)
+
+    const core = this
+    this._relayEvent = function (event, arg1, arg2, arg3) {
+      return core.emitAsync(event, this, arg1, arg2, arg3)
```

Comment:
I think this should be done with the `arguments` keyword instead of picking a certain number of args